zeebe/protocol/src/main/resources/protocol.xml

```diff
@@ -76,6 +76,7 @@
       <validValue name="MULTI_INSTANCE">58</validValue>
       <validValue name="RUNTIME_INSTRUCTION">59</validValue>
       <validValue name="BATCH_OPERATION_INITIALIZATION">60</validValue>
+      <validValue name="GLOBAL_VARIABLE">61</validValue>
```

Comment:
🔧 IMO, we should avoid custom concepts in the code. Cluster variables are just variables at the cluster scope, so there should be no need for a new record value. We can reuse `VARIABLE_DOCUMENT` instead. Note that it will require us to consider a value for `updateSemantics`, which we can cover by choosing `LOCAL` (it's locally scoped to the cluster scope). This will enable future changes where tenant variables can be built on top of this, as we do not want propagation semantics from tenant to cluster scope.